public/app/features/explore/RichHistory/RichHistoryQueriesTab.tsx

```diff
@@ -197,7 +197,7 @@ export function RichHistoryQueriesTab(props: Props) {
               />
             </div>
           )}
-          <div className={styles.sort}>
+          <div className={styles.sort} aria-label="sort queries">
```

Comment:
Nitpick: majority of our aria-labels start with a capital letter. Shouldn't we do the same there?

Reply:
👍 Good catch! Capitalised.